src/main/java/com/ibm/devops/connect/CloudSocketComponent.java

```diff
@@ -210,11 +210,11 @@ public void handleDelivery(String consumerTag, Envelope envelope,
                             String encodedString = getEncodedString(plainCredentials);
                             String authorizationHeader = "Basic " + encodedString;
                             String rootUrl = Jenkins.getInstance().getRootUrl();
-                            HttpResponse<String> response = Unirest.get(rootUrl+"job/"+jobName.replaceAll(" ", "%20")+"/lastBuild/consoleText")
+                            HttpResponse<String> response = Unirest.get(rootUrl+"job/"+jobName.replaceAll("/", "/job/").replaceAll(" ", "%20")+"/lastBuild/consoleText")
```

Comment:
probably best to URL encode instead of manually trying to replaceAll for special chars

Reply:
Made a small change with URL, removed replacing whitespaces with '%20' as Unirest is handling whitespaces at itself. Using URL encoding will make it more complicated as URL encoding replaces whitespaces with the '+' sign and then we have to replace the '+' sign with '%20' manually.